homeassistant/components/owntracks/messages.py

```diff
@@ -105,7 +111,13 @@ def _set_gps_from_zone(kwargs, location, zone):
 def _decrypt_payload(secret, topic, ciphertext):
     """Decrypt encrypted payload."""
     try:
-        keylen, decrypt = get_cipher()
+        if supports_encryption():
+            keylen, decrypt = get_cipher()
+        else:
+            _LOGGER.warning(
+                "Ignoring encrypted payload because libsodium not installed"
```

Comment:
This doesn't mean that libsodium is not installed. If we don't support encryption it's beacuse `NaCl` is not installed (which uses libsodium under the hood). It's an optional requirement.